backends-velox/src/test/scala/io/glutenproject/execution/VeloxStringFunctionsSuite.scala

```diff
@@ -60,13 +60,75 @@ class VeloxStringFunctionsSuite extends WholeStageTransformerSuite {
       s"from lineitem limit $LENGTH") { checkOperatorMatch[ProjectExecTransformer] }
   }
 
-  ignore("day_of_month") {
+  test("extract") {
+    runQueryAndCompare(s"select l_orderkey, l_shipdate, " +
+      s"extract(doy FROM DATE'2019-08-12') " +
+      s"from lineitem limit $LENGTH") { checkOperatorMatch[ProjectExecTransformer] }
+  }
+
+  ignore("second") {
```

Comment:
Could you add a comment here to explain why this test is ignored for now?

Reply:
Done